src/backend/tests/it/config.rs

```diff
@@ -25,6 +26,18 @@ fn config_is_available_to_allowed_users_only() {
         update_call::<UserProfile>(&pic_setup, nns_root, "config", ()).is_err(),
         "NNS root should not be able to call config"
     );
```

Comment:
Here we check that controllers are able to make a protected API call.  There is an existing check that other users are NOT able to make this API call.